<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.sento.organisations.OrgServiceRestControllerUpdateTest" tests="7" skipped="0" failures="0" errors="0" timestamp="2019-01-29T17:44:03" hostname="DESKTOP-M81G07M" time="0.217">
  <properties/>
  <testcase name="updateOrgReturns404IfNotFound" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.024"/>
  <testcase name="get400OnUpdateIfEmailInvalidFormat" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.049"/>
  <testcase name="updateOrganisation" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.049"/>
  <testcase name="failUpdateIfEmailNull" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.019"/>
  <testcase name="get400OnUpdateIfContactNameNull" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.02"/>
  <testcase name="updateOrgReturns405IfOrgIdMismatch" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.02"/>
  <testcase name="updateOrganisationFromJson" classname="com.sento.organisations.OrgServiceRestControllerUpdateTest" time="0.027"/>
  <system-out><![CDATA[2019-01-29 17:44:03.156  INFO 20760 --- [    Test worker] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [com.sento.organisations.OrgServiceRestControllerUpdateTest], using SpringBootContextLoader
2019-01-29 17:44:03.157  INFO 20760 --- [    Test worker] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [com.sento.organisations.OrgServiceRestControllerUpdateTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2019-01-29 17:44:03.163  INFO 20760 --- [    Test worker] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
2019-01-29 17:44:03.163  INFO 20760 --- [    Test worker] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@15176421, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@3259806c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@62a56121, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@26559b17, org.springframework.test.context.support.DirtiesContextTestExecutionListener@8cdc590, org.springframework.test.context.transaction.TransactionalTestExecutionListener@76e96e05, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@3f8cb590, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@358b1c55, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@538cae31, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@6ea87b03, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@34d27373, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@5956b808]
2019-01-29 17:44:03.171  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.171  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.175  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 3 ms
2019-01-29 17:44:03.185  INFO 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Update for organisation in PUT for id BM099
2019-01-29 17:44:03.186 DEBUG 20760 --- [    Test worker] org.hibernate.SQL                        : select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
Hibernate: select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
2019-01-29 17:44:03.188 ERROR 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Attempted to update non-existent  org : BM099
2019-01-29 17:44:03.197  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.197  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.201  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 4 ms
2019-01-29 17:44:03.237  WARN 20760 --- [    Test worker] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [1] in public org.springframework.http.ResponseEntity<?> com.sento.organisations.controller.OrganisationServiceController.updateOrganisation(java.lang.String,com.sento.organisations.model.Organisation): [Field error in object 'organisation' on field 'contactEmail': rejected value [foo]; codes [Email.organisation.contactEmail,Email.contactEmail,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [organisation.contactEmail,contactEmail]; arguments []; default message [contactEmail],[Ljavax.validation.constraints.Pattern$Flag;@6eba4a3c,org.springframework.validation.beanvalidation.SpringValidatorAdapter$ResolvableAttribute@27b78ed8]; default message [must be a well-formed email address]] ]
2019-01-29 17:44:03.248  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.248  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.254  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 6 ms
2019-01-29 17:44:03.259  INFO 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Update for organisation in PUT for id BM023
2019-01-29 17:44:03.260 DEBUG 20760 --- [    Test worker] org.hibernate.SQL                        : select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
Hibernate: select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
2019-01-29 17:44:03.274 DEBUG 20760 --- [    Test worker] org.hibernate.SQL                        : update organisations set email=?, contact_name=?, contact_phone=?, name=?, org_id=? where id=?
Hibernate: update organisations set email=?, contact_name=?, contact_phone=?, name=?, org_id=? where id=?
2019-01-29 17:44:03.297  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.298  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.302  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 4 ms
2019-01-29 17:44:03.308  WARN 20760 --- [    Test worker] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [1] in public org.springframework.http.ResponseEntity<?> com.sento.organisations.controller.OrganisationServiceController.updateOrganisation(java.lang.String,com.sento.organisations.model.Organisation): [Field error in object 'organisation' on field 'contactEmail': rejected value [null]; codes [NotNull.organisation.contactEmail,NotNull.contactEmail,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [organisation.contactEmail,contactEmail]; arguments []; default message [contactEmail]]; default message [must not be null]] ]
2019-01-29 17:44:03.318  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.318  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.322  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 4 ms
2019-01-29 17:44:03.330  WARN 20760 --- [    Test worker] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [1] in public org.springframework.http.ResponseEntity<?> com.sento.organisations.controller.OrganisationServiceController.updateOrganisation(java.lang.String,com.sento.organisations.model.Organisation): [Field error in object 'organisation' on field 'contactName': rejected value [null]; codes [NotNull.organisation.contactName,NotNull.contactName,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [organisation.contactName,contactName]; arguments []; default message [contactName]]; default message [must not be null]] ]
2019-01-29 17:44:03.339  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.340  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.344  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 4 ms
2019-01-29 17:44:03.350  INFO 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Update for organisation in PUT for id BM023
2019-01-29 17:44:03.350 ERROR 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Update failed - path/body org id mismatch for BM099
2019-01-29 17:44:03.361  INFO 20760 --- [    Test worker] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2019-01-29 17:44:03.361  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2019-01-29 17:44:03.366  INFO 20760 --- [    Test worker] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 5 ms
2019-01-29 17:44:03.370  INFO 20760 --- [    Test worker] c.s.o.c.OrganisationServiceController    : Update for organisation in PUT for id BM023
2019-01-29 17:44:03.371 DEBUG 20760 --- [    Test worker] org.hibernate.SQL                        : select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
Hibernate: select organisati0_.id as id1_0_, organisati0_.email as email2_0_, organisati0_.contact_name as contact_3_0_, organisati0_.contact_phone as contact_4_0_, organisati0_.name as name5_0_, organisati0_.org_id as org_id6_0_ from organisations organisati0_ where organisati0_.org_id=?
2019-01-29 17:44:03.375 DEBUG 20760 --- [    Test worker] org.hibernate.SQL                        : update organisations set email=?, contact_name=?, contact_phone=?, name=?, org_id=? where id=?
Hibernate: update organisations set email=?, contact_name=?, contact_phone=?, name=?, org_id=? where id=?
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
